### DISCUSSION TITLE
SNSD: what is zero-parity erasure coding

### DISCUSSION THREAD

**Discussion by matwey** [minio/minio#17710] [Q&A] [answered]
Hi,

I am trying to figure out what are advantages of SNSD over deprecated FS backend. I see that the docs say that SNSD use "zero-parity erasure coded backend", but never explains what is it.

1. Does SNSD zero-parity erasure coding provide additional data protection comparing to FS?
2. Is there Bit Rot protection in SNSD?
3. Is SNSD just code unification backend allowing to use the same code for SNSD and MNMD?

**Comment by harshavardhana** [comment, marked as answer]
SNSD supports all the features that a multi drive deployment supports which couldn't be supported via FS mode.

Yes there is bitrot protection in SNSD.

**Reply by klauspost** [reply]
"bitrot detection" is probably the most correct term.
